src/main.go

```diff
@@ -22,7 +22,7 @@ func NewTemporalCloudProvider(temporalCloudAdapter ITemporalCloudAdapter, config
 
 	err := temporalCloudAdapter.Flags().Parse(os.Args)
 	if err != nil {
-		return nil, fmt.Errorf("unable to parse flags: %v", err)
+		return nil, fmt.Errorf("failed to parse flags: %v", err)
```

Comment:
Changing the error message wording to 'failed to parse flags' aligns the language more closely with conventional error handling messages. Ensure this pattern is consistent across similar error handling scenarios.